app/calendar/index.tsx

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
```

Comment:
Accidentally committed this file. Will put it into a separate PR before merging this one.